tests/libdv/dv.go

```diff
@@ -168,46 +189,58 @@ func PVCWithStorageClass(sc string) pvcOption {
 	}
 }
 
-// PVCWithVolumeSize overrides the default volume size (cd.CirrosVolumeSize), with the size parameter
+// Storage Options
+// StorageWithStorageClass add the sc storage class name to the DV
+func StorageWithStorageClass(sc string) storageOption {
+	return func(storage *v1beta1.StorageSpec) {
+		if storage == nil {
+			return
+		}
+
+		storage.StorageClassName = &sc
+	}
+}
+
+// StorageWithVolumeSize overrides the default volume size (cd.CirrosVolumeSize), with the size parameter
 // The size parameter must be in parsable valid quantity string.
-func PVCWithVolumeSize(size string) pvcOption {
-	return func(pvc *corev1.PersistentVolumeClaimSpec) {
-		if pvc == nil {
+func StorageWithVolumeSize(size string) storageOption {
+	return func(storage *v1beta1.StorageSpec) {
+		if storage == nil {
```

Comment:
maybe in this case where someone requests to update storage param and the storage is nil its better to print some warning, cause it might be hard for someone to understand why something is failing only cause it used the wrong function to change the volume size

Reply:
same for everywhere where we silently return if storage/pvc is nil

Reply:
I don't want to change how that works to be honest, I think it makes sense to simply fail here, but that would be significantly changing the behavior that currently exists.

Reply:
It doesnt really changes anything, just add a log before the return nil..